woodwork/tests/utils/test_accessor_utils.py

```diff
@@ -38,6 +38,22 @@ def test_init_series_valid_conversion_specified_ltype(sample_series):
     assert series.ww.semantic_tags == set()
 
 
+def test_init_series_with_pd_extension_array():
+    extension_categories = pd.Categorical([1, 2, 3])
+    series = init_series(extension_categories)
+    series2 = init_series(pd.Series([1, 2, 3], dtype='category'))
```

Comment:
It might be nice to have a more descriptive variable name for `series2`. Maybe something like `extension_series` that way the assertions are a little more clear on why we want to compare the two series